exercises/L-functions-nested/exercise.js

```diff
@@ -1,5 +1,18 @@
+function greetings(mentor) {
 var mentor1 = "Daniel";
-var mentor2 = "Irina";
-var mentor3 = "Mimi";
-var mentor4 = "Rob";
-var mentor5 = "Yohannes";
+var mentor2 = "Chizim";
+var mentor3 = "Biruk";
+var mentor4 = "Louis";
+var mentor5 = "Ali";
+var mentor6 = "Marta";
+var mentor7 = "Elizabeth";
+var mentor8 = "Huma";
+var mentor9 = "Anna";
+var mentor10 = "Matt";
```

Comment:
These variables are not being used - do they need to be in here?
Should they perhaps be being used to call the function?